arviz/plots/backends/matplotlib/lmplot.py

```diff
@@ -0,0 +1,44 @@
+"""Matplotlib plot linear regression figure."""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from ...hdiplot import plot_hdi
+
+
+def plot_lm(x, y, y_model, y_ppc, num_pp_samples, kind_pp, kind_model):
+    """Matplotlib Linear Regression."""
+    _, ax = plt.subplots(1, 1, figsize=(6, 4), dpi=100)
```

Comment:
This could call the function in plot_utils. I think other plots use that func too.

Reply:
I have done it in my local branch. I will soon push the changes